### PR TITLE
Fix capitalization in terminal output

### PR DESCRIPTION
**Pull request by victorwuky**
Clubhouse story: https://app.clubhouse.io/windmill/story/4697/fix-capitalization-in-tui-output

### Before
before

### After
after

### CODE REVIEW

**Comment by victorwuky** [comment]
@maiamcc : Is this good? (I'll be sure to squash the commits before merging if everything else looks good.)

File: vendor/github.com/spf13/cobra/command.go [hunk 1]
```diff
@@ -967,7 +967,7 @@ func (c *Command) validateRequiredFlags() error {
 func (c *Command) InitDefaultHelpFlag() {
 	c.mergePersistentFlags()
 	if c.Flags().Lookup("help") == nil {
-		usage := "help for "
+		usage := "Help for "
```
Comment (maiamcc) [inline vendor/github.com/spf13/cobra/command.go]: Don't edit files in `vendor/`, they're external packages we depend on , and edits here will cause weirdness if/when we pull down the latest version of this package.

File: vendor/github.com/windmilleng/wmclient/pkg/analytics/cli.go [hunk 2]
```diff
@@ -51,7 +51,7 @@ const choiceFile = "analytics/user/choice.txt"
 func NewCommand() *cobra.Command {
 	analytics := &cobra.Command{
 		Use:   "analytics",
-		Short: "info and status about windmill analytics",
+		Short: "Info and status about windmill analytics",
 		RunE:  analyticsStatus,
 	}
```
Comment (maiamcc) [inline vendor/github.com/windmilleng/wmclient/pkg/analytics/cli.go]: See above re: `vendor/`

File: vendor/github.com/spf13/cobra/command.go (hunk 1 above)
Comment (victorwuky) [reply vendor/github.com/spf13/cobra/command.go]: Fixed thanks!

File: vendor/github.com/windmilleng/wmclient/pkg/analytics/cli.go (hunk 2 above)
Comment (victorwuky) [reply vendor/github.com/windmilleng/wmclient/pkg/analytics/cli.go]: Fixed thanks

**Pull request merged**